.github/workflows/test_release_version.yml

```
@@ -1,53 +1,53 @@
-# name: "Test Release version"
-
-# on:
-#   push:
-#     branches:
-#       - 'release/**'
-
-# env:
-#   OTP_VERSION: 23.3
-#   ELIXIR_VERSION: 1.11.4
-#   PHOENIX_VERSION: 1.5.7
-#   MIX_ENV: test
-
-# jobs:
-#   release_version_test:
-#     runs-on: ubuntu-latest
-
-#     steps:
-#       - name: Cancel Previous Runs
-#         uses: styfle/cancel-workflow-action@0.8.0
-#         with:
-#           access_token: ${{ github.token }}
-
-#       - uses: actions/checkout@v2
-#         with:
-#           ref: ${{ github.head_ref }}
-
-#      - uses: erlef/setup-elixir@v1
-#        with:
-#          otp-version: ${{ env.OTP_VERSION }}
-#          elixir-version: ${{ env.ELIXIR_VERSION }}
-
-#       - name: Cache Elixir build
-#         uses: actions/cache@v2
-#         with:
-#           path: |
-#             _build
-#             deps
-#           key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
-#           restore-keys: |
-#             ${{ runner.os }}-mix-
-
-#       - name: Install Dependencies
-#         run: mix deps.get
+name: "Test Release version"
```

The `"` character here isn't required. I suggest removing it to keep the consistency because we don't use it in the names of the steps
```suggestion
name: Test Release version
```

---

Adjusted in https://github.com/nimblehq/elixir-templates/pull/110/commits/1520f628b213a6613962dcc8eff58338da808cb0 thanks @longnd